### ISSUE TITLE
[Crypto] Have WOWNERO available at launch

### ISSUE THREAD

**Issue by nahuhh** [haveno-dex/listing#11]
- [x] I confirm there are no other issues about this asset (open or closed)
- [x] I understand that Haveno expects the project applying for being listed to take care of the PR for the actual implementation
- [x] I understand that adding the asset is at discretion of the Haveno Core Team and applying doesn't necessarily imply acceptance

### What's the ticker symbol of the project?

WOW

### Why should it be added to Haveno? 
![QkNzplGtRqVL2EGx-MonerovsWownero.jpg](https://user-images.githubusercontent.com/50635951/165939718-87e2760a-4c55-4225-8a47-02e65aca4886.jpg)

How would it benefit the project and the community?
- Suspected high volume pair due to
- thin competition from other exchanges 
- xmr <-> wow 

### Is it an Ethereum token (ERC-20)?

- [ ] Yes
- [x] No

### Was the coin pre-mined before public launch?

- [ ] Yes
- [x] No

### Is there a dev fee for the project maintainers?

- [ ] Yes
- [x] No

### Please provide resources about the project (Website, Twitter, etc)

You can find more info about WOW in your heart.

..also on wownero.org
Suchwow.xyz for documentation
### Additional notes

If WOW writes the pr, might just ve a meme. Better off doing it yourself.

**Comment by plowsof** [comment]
WOW rpc calls are pretty much 'like for like' to Moneros' . The only difference i have found is that their atomic units (xmr = 12, wow=11) and currently it has a trading pair on [Tradeogre](https://tradeogre.com/exchange/BTC-WOW) (wow-btc)

**Comment by nahuhh** [comment]
Yes. Trade ogre is the only "real" competitor and does 10-20k in volume daily.

With a market cap of 7m.

Thats a _lot_ more than most coins on bisq, including big names ltc, doge, zcash, etc

Wow could be the top performing coin haveno.

Nice gentle intro to privacy.

Edit: top volume* crypto, aside from btc and eth and xyz stablecoin

**Comment by HavenoDEX** [comment]
Please edit the tile so it's consistent with the provided template

> Why should it be added to Haveno?

This question hasn't been answered. Please don't use memes and try to be as clear as possible.

**Comment by HavenoDEX** [comment]
@nahuhh this part breaks the template. Please fix:

```
### Why should it be added to Haveno? 
![QkNzplGtRqVL2EGx-MonerovsWownero.jpg](https://user-images.githubusercontent.com/50635951/165939718-87e2760a-4c55-4225-8a47-02e65aca4886.jpg)

How would it benefit the project and the community?
- Suspected high volume pair due to
- thin competition from other exchanges 
- xmr <-> wow 

```

**Comment by nahuhh** [comment]
@HavenoDEX 

Did that fix it?

**Comment by plowsof** [comment]
Don't forget they enforce solo mining, Monero has centralised mining pools (p2pool will eventually fix this) https://pools.xmr.wiki/

**Comment by HavenoDEX** [comment]
> Did that fix it?

Yes. Thank you.

**Comment by SamsungGalaxyPlayer** [comment]
My blunt thoughts are below:

1. The entire Wownero userbase also uses Monero. Wownero users are exclusively a niche of Monero users. Adding Wownero won't add more users to Haveno.

2. There's no incremental utility. There are no exchanges that list WOW or merchant's who accept WOW who don't also list/accept XMR.

This inclusion benefits WOW users, but not XMR users (those who aren't also WOW users) to a significant extent.

Still, supporting coins overall is a good thing. Any additional volume is a good thing for Haveno and Monero.

Thus, I recommend this be considered for the second or third stage of listing assets. Adding stables will have a far more meaningful impact imo to both Haveno and Monero.

**Comment by nahuhh** [comment]
> My blunt thoughts are below:
> 
>  Adding Wownero won't add more users to Haveno.

Should add volume. Which is good for monero/haveno (fees)

> 2. There's no incremental utility. There are no exchanges that list WOW or merchant's who accept WOW who don't also list/accept XMR.
> 

WOW+ xmr users dont want to use a cex or go through btc in order to swap.

I believe they would move to haveno, even with higher fees

> This inclusion benefits WOW users, but not XMR users (those who aren't also WOW users) to a significant extent.
> 

Nobody holds wow for any amount of time, it gets funneled into (I assumed) xmr. 

> Still, supporting coins overall is a good thing. Any additional volume is a good thing for Haveno and Monero.
> 

I think wow would be a top pair, aside from stables, btc and eth.

> Thus, I recommend this be considered for the second or third stage of listing assets. Adding stables will have a far more meaningful impact imo to both Haveno and Monero.

I agree with stables, but I think WOW should be on release for a few reasons

- the only private stable has Dev fees and dev mint fees (xUSD).
Note: I fully support adding xUSD, ~~UST~~, DAI. Not sure if usdt or usdc are a good idea.
- day 1 volume
- userbase would be pleased
- if wow grows, xmr should stand to be the major beneficiary 

Disclaimer: I own 0 wow. But I would play with it on haveno over any public blockchain.

**Comment by equation-group** [comment]
Please don't waste yours and community's time adding useless shit to haveno,  close these similar proposals request.

**Comment by nahuhh** [comment]
@equation-group 

What is your definition of useless?
I think a major metric for an exchange should be volume, and competing for said volume. 

Some pairs on bisq do quite well. Notably xmr btc and doge.

Litecoin and zcash do 0 volume.
0 bids and 0 asks, 0 liquidity, makes an exchange useless.

Doge might be trash, but IMO it makes more sense to list doge than litecoin.
It actually has liquidity.
(Im not suggesting pushing Litecoin back) 
Bisq:
![Image_101.jpg](https://user-images.githubusercontent.com/50635951/166057618-06af0d83-4433-4a74-9f70-e079e3acae3a.jpg)

**Comment by XfedeX** [comment]
Wownero is the first privacy-respecting memecoin.
It's actually more useful than most "serious" coins. Its main usecase is microtipping memes on suchwow.xyz .
I think that Wownero is a "privacy trojan horse", as it teaches people about privacy and Monero using memes.

**Comment by BugBud** [comment]
@SamsungGalaxyPlayer @equation-group 

I understand where you are coming from I once too thought that WoW did not bring anything new to the table however I have come to change my view in this respect. 

1. Wownero appeals to a different kind of person than Monero, someone less focused on Privacy and more on memes. 

2. Wownero really needs Haveno that is Monero has multiple exchanges but Wownero users would really embrace a Wownero listing because it is on just one exchange with reliable volume and if Wownero was added by Haveno I am sure that the community will bring its incredible meme and art skills to bear on Haveno which will be a fun push for Haveno.

3. Wownero has a long distribution ( 50 years) this isnt so relevant to those of us that have been mining monero for years but to new miners so much of the Monero is already mined and I truly believe billions of people could come to mine Monero and some might ask why did such a small group get such a high concentration of the mined coins. If people in 20 years decide yes CPU mining is the future and I should do it they might realize they are just getting a portion of the tail emission.  With Wownero they wont be just getting tail emission but there will still be a significant percent of the total available for grabs for new cpu miners in the next few decades. This doesnt take anything away from Monero it just suggests that when the masses adopt cpu mining they might find the relatively short emission curve of Monero unfair to them and want to participate in a flatter emission crypto like Wownero. Its just taking into account the long learning curve for adopting fairly mined asic resistant privacy coins. 

4.  There are so many erc20 cryptos and meme coins that to compete with its model Monero has to have some clones, of the 9000 cryptos less than 300 have a blockchain, of those less than 100 are POW and of those less than 50 are pure POW fair launch coins of those a handful implement privacy solutions. 

5. Wownero has a distinctly different community, one great example of this is Wownero implemented the first VR wallet and that VR compatible wallet overlay was used extensively by DJs in virtual parties in VR chat and NEOS to tip the DJS and dancers.  The meme community on suchwow.xyz is another great example of uniquely wownero users.  Youll notice on coinmarketcap Wownero has several tags that Monero does not such as Meme coin and VR/AR coin. 

6. Finally Wownero is a fully community effort there's no free coins for development its not easy running everything on bug bounties and through donations but it creates a fair ecosystem there are a large number of XMR clones but most have premines, dev funds and marketing budgets. WOW operates like Bitcoin and Monero without any unmined free coins for anyone and thats a difficult thing to do when devs can make big money on the latest instamined project so any help and support from the Haveno team is greatly appreciated. Many exchanges want big bucks to list WoW so getting it on Haveno would mean the world to the community. 

If Haveno moves forward Id be willing to create a Wow bounty for the team on the Wownero funding website. 

Thanks

**Comment by HavenoDEX** [comment]
This is not a forum, please stop posting memes here. They clutter the screen with useless information and make harder to follow the discussion. We have removed all memes except the one in the OP.

**Comment by dginovker** [comment]
While I agree that Wownero brings little measurable value to the table, it's an honest cryptocurrency with a development team that in this bear market, has proven the test of time. It's a breath of fresh air in the cryptospace, and as of writing not a bad cryptocurrency for Haveno to be associated with.

Certainly stable and high volume coins are better, but it's great to give and have fun too. I certainly support this proposal :+1:

**Comment by ghost** [comment]
I agree with @dginovker 's sentiment

**Comment by jwinterm** [comment]
>     1. The entire Wownero userbase also uses Monero. Wownero users are exclusively a niche of Monero users. Adding Wownero won't add more users to Haveno.

Anecdotally, I know at least a few WOW users that don't touch XMR, so probably correct to say "almost the entire Wownero userbase", but not quite at least, and just because someone "uses" both doesn't mean they don't want a means to swap in between the two.
